model/host/db.go

```diff
@@ -1079,3 +1079,13 @@ func FindOneVolume(query interface{}) (*Volume, error) {
 	}
 	return v, err
 }
+
+func FindVolumesByUser(userID string) ([]*Volume, error) {
+	volumes := []*Volume{}
```

Comment:
In general we try to avoid slices of pointers if possible, because it's easy to get `range` statements over them wrong. Can this be a slice of `Volume`s instead, instead of `*Volume`s?

Reply:
Sounds good